pstoreds/addr_book.go

```diff
@@ -79,141 +98,206 @@ func (mgr *dsAddrBook) AddAddrs(p peer.ID, addrs []ma.Multiaddr, ttl time.Durati
 	if ttl <= 0 {
 		return
 	}
-
-	mgr.setAddrs(p, addrs, ttl, true)
+	mgr.setAddrs(p, addrs, ttl, false)
```

Comment:
Changed the role of the last argument from `add` to `ttlReset`, which is clearer and illustrates the behaviour.